examples/split-screen/layouts/new-scene.json

```diff
@@ -0,0 +1,1020 @@
+{
```

Comment:
Is this scene used? If not, please remove it :)

Reply:
Ah, pesky GDevelop not deleting old scene files when renaming them